docs/cedar.md

```diff
@@ -38,40 +38,24 @@ However, you can click Alfabet on the main CEDAR UI page to see the Alfabet UI,
 
 ## How is CEDAR accessed within MINT?
 
-The calls that are made to the CEDAR APIs are defined by code generated by `go-swagger`. However, in order to not have changes to the CEDAR API documentation break MINT, a "translated client" is usually created for each CEDAR API that wraps the generated code with our own, tested client. An example of this is the [pkg/cedar/intake/client.go](../pkg/cedar/intake/client.go) file.
+The calls that are made to the CEDAR APIs are defined by code generated by `go-swagger`. However, in order to not have changes to the CEDAR API documentation break MINT, a "translated client" is usually created for each CEDAR API that wraps the generated code with our own, tested client. An example of this is the [pkg/cedar/cedarldap/gen/client/l_d_a_p_client.go](../pkg/cedar/cedarldap/gen/client/l_d_a_p_client.go) file.
```

Comment:
For the translated client, we should point to `pkg/cedar/cedarldap/translated_client.go`, rather than the file in the `/pkg/cedar/cedarldap/gen/` directory